dir.props

```diff
@@ -145,17 +140,22 @@
   <!-- .NET Core build support -->
   <PropertyGroup Condition="'$(NetCoreBuild)' != 'true'">
     <TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v4.5.1</TargetFrameworkVersion>
-    <PlatformTarget>x86</PlatformTarget>
     <BaseNuGetRuntimeIdentifier>win7</BaseNuGetRuntimeIdentifier>
   </PropertyGroup>
   <PropertyGroup Condition="'$(NetCoreBuild)' == 'true'">
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
-    <PlatformTarget>x86</PlatformTarget>
```

Comment:
Actually is there any reason that we cannot use AnyCPU for these task libraries? We are having any issue right now for example where we are trying to build and AnyCPU tasks library that we can use for both windows and linux but we cannot because of this split.